README.adoc

```diff
@@ -1 +1,53 @@
-= bisq network roles
+= Bisq Network Roles
+
+This repository gives you a transparent overview about all current roles within the Bisq Network. It provides the viewer with details about the purpose, motivation, privileges, requirements and responsibilities of all roles. This will ensure clarity about ones own role, help identifying other roles and facitilitate the compensation process.
```

Comment:
I'd yank the word 'transparent' here, sounds a little buzzwordy. Maybe just 'complete overview' instead.

 - ones => one's
 - typo in facilitate